gulp/config.js

```diff
@@ -83,7 +69,57 @@ module.exports = (function() {
     var options = {
       files: [].concat(
         './node_modules/phantomjs-polyfill/bind-polyfill.js',
-        wiredep({devDependencies: true}).js,
+        './node_modules/jquery/dist/jquery.js',
```

Comment:
This is problematic ;). Is there any way to make `wiredep` work? Is it just that wiredep can't read package.json or pick the right libraries, or is it also that those don't have the necessary info on what to include?

People will notice that adding the lib is not enough and add it to javascripts, but.. these test deps will get out of sync soon, I'm afraid..